src/ports/postgres/dbconnector/TypeTraits_impl.hpp

```diff
@@ -750,6 +784,30 @@ struct TypeTraits<dbal::eigen_integration::SparseColumnVector> {
     );
 };
 
+template <>
+struct TypeTraits<std::vector<uint16_t> > {
+    typedef std::vector<uint16_t> value_type;
+
+    WITH_OID( INT4ARRAYOID );
+    WITH_MUTABILITY( dbal::Mutable );
+    WITH_DEFAULT_EXTENDED_TRAITS;
+    // FIXME a quick hack
```

Comment:
We ended up not using this new functionality (using double ptr instead).  I will remove this code, we can always add it back later.  Does that seem reasonable?